ts/examples/vite-browser-render/src/App.tsx

```diff
@@ -5,7 +5,7 @@ import MP4Player from './examples/MP4Player';
 
 const EXAMPLES = {
   'counter': <Counter />,
```

Comment:
Do not define that globally, keep that iniside the App component to make sure that freeing stuff is correct.